app/assets/stylesheets/application/bootstrap-classes-overrides.scss

```diff
@@ -0,0 +1,7 @@
+.btn-primary {
+  @extend .text-white;
+}
+
+.bg-info.badge {
+  color: $primary;
+}
```

Comment:
Can't we use sass vars to tweaks those things?

Reply:
we have this variable `$badge-color` but it  would change the color for all badges and in `badge bg-secondary` we want it white